src/machine/machined-dbus.c

```diff
@@ -539,6 +539,10 @@ static int method_copy_machine(sd_bus_message *message, void *userdata, sd_bus_e
         return redirect_method_to_machine(message, userdata, error, bus_machine_method_copy);
 }
 
+static int method_copy_machine_with_flags(sd_bus_message *message, void *userdata, sd_bus_error *error) {
+        return redirect_method_to_machine(message, userdata, error, bus_machine_method_copy);
+}
```

Comment:
Do we actually need this? I figure we can just reuse `method_copy_machine()` no?